app/Http/Controllers/ImageController.php

```diff
@@ -3,11 +3,41 @@
 namespace App\Http\Controllers;
 
 use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+
+use View;
 
 use App\Image;
+use App\Content;
+use Validator;
+use App\Http\Controllers\ContentController;
+
 
 class ImageController extends Controller
 {
 
+	public function store(Request $request)
+    {
+        $v = Validator::make($request->all(), [
+            'image' => 'required|image'
+        ]);
+
+        if ($v->fails())
+        {
+            return redirect()->back()->withErrors($v->errors());
+        }
+
+        //save
+        $cc = new ContentController();
```

Comment:
Do not call controllers from other controllers. move storeImage to Image model or smth.